File: 3455-minimum-length-of-string-after-operations/minimum-length-of-string-after-operations.java
class Solution {
    public int minimumLength(String s) {
        int[] freq = new int[26];
        for (int i = 0; i < s.length(); i++)
            freq[s.charAt(i) - 'a']++;
        int ans = s.length();
        for (int i = 0; i < s.length(); i++) {
            if (freq[s.charAt(i) - 'a'] >= 3) {
                ans -= 2;
                freq[s.charAt(i) - 'a'] -= 2;
            }
        }
        return ans;
    }
}

File: 3455-minimum-length-of-string-after-operations/minimum-length-of-string-after-operations.cpp
class Solution {
public:
    int minimumLength(string s) {
        unordered_map<char,int>freq;
        for(auto c : s)
            freq[c]++ ;
        int n = s.size();
        for(int i : s){
            if(freq[i] >= 3){
                n -= 2;
                freq[i] -= 2;
            }
        }
        return n ;
    }
};